storage/types.go

```diff
@@ -153,8 +153,11 @@ type Namespace interface {
 	// NumSeries returns the number of series in the namespace
 	NumSeries() int64
 
-	// Shards returns the shards
+	// Shards returns the shard as storange shards
 	Shards() []Shard
+
+	// GetOwnedShards returns the database shards
+	GetOwnedShards() []databaseShard
```

Comment:
can this be part of `databaseNamespace` instead of `Namespace`?

Reply:
+1